.travis.yml

```diff
@@ -42,11 +42,6 @@ install:
 - pushd $TRAVIS_BUILD_DIR
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
```

Comment:
I wonder if sourcing setup_gae.sh may need to be added to setup.sh, but that's another issue for another time.